test/functional/vimscript/system_spec.lua

```diff
@@ -660,4 +660,20 @@ describe('shell :!', function()
         1]])
     end
   end)
+
+  it(':{range}! with powershell without redirecting to buffer', function()
+    local screen = Screen.new(500, 10)
+    screen:attach()
+    helpers.set_shell_powershell(true)
+    insert([[
+      3
+      1
+      4
+      2]])
+    feed(':4verbose %w !sort<cr>')
+    screen:expect{
+      any=[[Executing command: .?& { Get%-Content .* | & sort }]]
+    }
+    feed('<CR>')
```

Comment:
to troubleshoot the hanging CI, try adding `command('qa!')` here.

if that doesn't work, try commenting-out the test above this one.

Reply:
That solved the problem.  Thanks.

Reply:
I'm confused. How come this passed on Windows without it but not on Linux?